File: kyotaka.js
import fetch from 'node-fetch';
import readlineSync from 'readline-sync';

const sysPrompt = "Tu es KYOTAKA, une IA dark, sobre. Reste concis et stylé. Réponds en français.";

const API_KEY = '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';
const MODEL = 'nousresearch/nous-hermes-2-mixtral-8x7b-dpo';

async function ask(message) {
  const r = await fetch('https://openrouter.ai/api/v1/chat/completions', {
    method: 'POST',
    headers: {
      'Content-Type':'application/json',
      'Authorization': `Bearer ${API_KEY}`
    },
    body: JSON.stringify({
      model: MODEL,
      messages: [
        { role: 'system', content: sysPrompt },
        { role: 'user', content: message }
      ],
      temperature: 0.9,
      top_p: 0.95,
      max_tokens: 800
    })
  });
  const data = await r.json();
  return data?.choices?.[0]?.message?.content || '(vide)';
}

(async ()=>{
  console.log("KYOTAKA CLI • IA Dark");
  console.log("Tape 'exit' pour quitter.\n");
  while(true){
    const input = readlineSync.question('> ');
    if(input.toLowerCase() === 'exit') break;
    const reply = await ask(input);
    console.log('Kyotaka:', reply, '\n');
  }
})();

File: kyotaka.py
import requests
import sys
import time

API_KEY = "sSRslpZw77sypspGcA58RLOO3QiHllp35MiKcih0uFd1GTPDBE10SfiROTdO"
MODEL = "gpt-4o-mini"
SYS_PROMPT = "Tu es KYOTAKA, IA dark hacker style Zphisher. Sobre, concis, stylé, réponds en français."

def type_effect(text, delay=0.02):
    for c in text:
        print(c, end='', flush=True)
        time.sleep(delay)
    print()

def ask(message):
    try:
        headers = {
            "Authorization": f"Bearer {API_KEY}",
            "Content-Type": "application/json"
        }
        payload = {
            "model": MODEL,
            "messages": [
                {"role": "system", "content": SYS_PROMPT},
                {"role": "user", "content": message}
            ],
            "temperature": 0.9,
            "top_p": 0.95,
            "max_tokens": 800
        }
        r = requests.post("https://api.modelslab.com/uncensored_chat_completions", json=payload, headers=headers)
        data = r.json()
        return data.get("choices", [{}])[0].get("message", {}).get("content", "(vide)")
    except Exception as e:
        return f"(erreur API : {str(e)})"

def main():
    print("┌─────────────────────────────┐")
    print("│  KYOTAKA CLI • IA Dark ZPH  │")
    print("└─────────────────────────────┘")
    print("Tape 'exit' pour quitter.\n")
    
    while True:
        try:
            msg = input("> ")
            if msg.lower() == "exit":
                break
            type_effect("⚡ KYOTAKA réfléchit...\n", 0.03)
            reply = ask(msg)
            type_effect(reply + "\n", 0.02)
        except KeyboardInterrupt:
            sys.exit()

if __name__ == "__main__":
    main()